contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -5,22 +5,22 @@ import (
 	"errors"
 	"flag"
 	"fmt"
-	"log/slog"
-	"net/http"
 	"os"
 	"path/filepath"
-	"time"
 
-	"github.com/gnolang/gno/contribs/gnodev/pkg/address"
-	gnodev "github.com/gnolang/gno/contribs/gnodev/pkg/dev"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/emitter"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/rawterm"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/watcher"
+	"github.com/gnolang/gno/contribs/gnodev/pkg/packages"
 	"github.com/gnolang/gno/gno.land/pkg/integration"
 	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
 	"github.com/gnolang/gno/tm2/pkg/commands"
 	"github.com/gnolang/gno/tm2/pkg/crypto"
-	osm "github.com/gnolang/gno/tm2/pkg/os"
+)
+
+const DefaultDomain = "gno.land"
+
+var (
```

Comment:
const?

Reply:
I was keeping them to be able to bundle them into one declaration since `MustAddressFromString(...)` cannot be a const, but I don't mind changing it: 5de07a88b12faffebfd32a4a471d3b0644b7f796